docs/release-notes/Percona-Everest-1.8.0-(2025-06-29).md

```diff
@@ -0,0 +1,131 @@
+    | **5.**|[Known limitations](https://docs.percona.com/everest/release-notes/Percona-Everest-1.7.0-%282025-05-29%29.html#known-limitation)|Discover all the known limitations in Percona Everest 1.8.0|
+
+
+## 🌟 Release highlight
+
+### Import external backups into Percona Everest clusters
+
+!!! warning
+    The **external backup import** feature in Percona Everest is currently in **Technical Preview**. Early adopters are advised to use this feature only for testing purposes and **not in production environments**.
+
+Starting with Percona Everest 1.8.0, we are excited to roll out a new feature that allows you to directly import database backups from external object storage into clusters managed by Percona Everest. This feature leverages an extensible framework that simplifies your backup process.
+
+#### Key features
+
+- Import database backups from external storage, such as Amazon S3, directly into database clusters managed by Percona Everest.
+
+- Customize the import process using tools like `mongodump`, `pg_dump`, or `mysqldump`.
+
+- A pluggable and extensible framework that can adapt to various import needs and workflows.
+    
+#### How to import external database backups using the Percona Everest UI
+
+Here are the steps to import the external backups:
+{.power-number}
+
+1. Navigate to the Percona Everest homepage and click **Import**.
+    
+2. Select the database type you want to import (MySQL, PostgreSQL, or MongoDB). The **Basic information** page will be displayed.
+
+3. Fill in the details on the **Basic information** page and click **Continue**. This will take you to the **Import information** page.
+
+4. On the **Import Information** page, enter the following:
+
+    - Choose the data importer from the dropdown.
+
+    - Provide S3 details.
+
+        ![!image](../images/mongodb_s3_details_importers.png)
+
+    -  Specify the **File Directory** path within your S3 bucket where the backup files are stored. Click **Save**.
+
+        Refer to the [documentation](https://docs.percona.com/everest/backups_and_restore/how_to_import_backups.html) for details on the correct file path format.
+
+        ![!image](../images/importers_mongo_file_path.png)
+
+    -  Enter the key-value pairs and user secrets (For MongoDB and MySQL) in the **DB credentials** section.
+
+        ![!image](../images/importers_mongodb_db_credentials.png)
+
+5. Click **Continue** until you reach the end of the wizard. You will be notified once the import is successfully completed.
+
+
+## New features
+
+- [EVEREST-2068](https://perconadev.atlassian.net/browse/EVEREST-2068), [EVEREST-2069](https://perconadev.atlassian.net/browse/EVEREST-2069), [EVEREST-2070](https://perconadev.atlassian.net/browse/EVEREST-2070): Starting with Percona Everest 1.8.0, you can now import external backups from PostgreSQL, MySQL, and MongoDB, respectively, into clusters managed by Percona Everest using a simplified and extensible DataImporter framework.
```

Comment:
Please feel free to improve this suggestion .. My point is that we should mention clearly that the default data importers work for the Percona operators only, but the underlying framework allows you to customize your import process ...

```suggestion
- [EVEREST-2068](https://perconadev.atlassian.net/browse/EVEREST-2068), [EVEREST-2069](https://perconadev.atlassian.net/browse/EVEREST-2069), [EVEREST-2070](https://perconadev.atlassian.net/browse/EVEREST-2070): Starting with Percona Everest 1.8.0, you can now import external backups taken using the Percona PostgreSQL, MySQL, and MongoDB operators into clusters managed by Percona Everest. Additionally, the newly added Data Importers framework allows you to customize your import process using backup/restore tools of your choice.
```